Adding ETag in the response header

adriane-cardozo opened the pull request:
Fixed #160 

Adding ETag in the response header for computer system and chassis.

ffdarkpenguin commented on oneview_redfish_toolkit/tests/blueprints/test_chassis.py:
```diff
@@ -159,6 +159,34 @@ def test_get_enclosure_chassis(
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
+        self.assertEqual("application/json", response.mimetype)
+        self.assertEqual(self.enclosure_chassis_mockup, json_str)
+        self.assertEqual(
```
I would just add this assert line to the enclosure_chassis test. This work, is OK but the other is optimized in my opinion. Same for the other tests.

ricardoas replied:
I kind of agree with this... as there should not be a enclosure chassis without an etag from now on.
Can you update that, @adriane-cardozo?

adriane-cardozo commented:
@ffdarkpenguin ok, I have just updated the unit tests, moving ETag test to the existing unit tests.
@ricardoas ok, I have just added ETag for Managers and updated unit tests.
